onset_maproom/layout.py

```diff
@@ -438,50 +434,57 @@ def map_layout(center_of_the_map):
                 id="map",
                 center=center_of_the_map,
                 zoom=CONFIG["zoom"],
+                maxBounds = [[-5.790897,25.839844],[17.978733,61.523438]],
+                minZoom = CONFIG["zoom"] - 1,
+                #maxZoom = CONFIG["zoom"] + 10, #this was completely arbitrary
                 style={
                     "width": "100%",
-                    "height": "50vh",
+                    "height": "100%",#height of the map 
                 },
             ),
             html.H6(
                 id="map_title"
             )
         ],
         fluid=True,
-        style={"padding": "0rem"},
+        style={"padding": "0rem", "height":"90%"},#box that holds map and title
     )
 
 
 def results_layout():
-    return dbc.Tabs(
-        [
-            dbc.Tab(
-                [
-                    html.H6(id="germination_sentence"),
-                    dbc.Spinner(dcc.Graph(id="onsetDate_plot")),
-                    dbc.Spinner(dcc.Graph(id="probExceed_onset")),
-                ],
-                label="Onset Date",
-            ),
-            dbc.Tab(
+    return html.Div( 
+        [   
+            dbc.Tabs(
                 [
-                    dbc.Spinner(dcc.Graph(id="cessDate_plot")),
-                    dbc.Spinner(dcc.Graph(id="probExceed_cess")),
+                    dbc.Tab(
+                        [
+                            html.H6(id="germination_sentence"),
+                            dbc.Spinner(dcc.Graph(id="onsetDate_plot")),
+                            dbc.Spinner(dcc.Graph(id="probExceed_onset")),
+                        ],
+                        label="Onset Date",
+                    ),
+                    dbc.Tab(
+                        [
+                            dbc.Spinner(dcc.Graph(id="cessDate_plot")),
+                            dbc.Spinner(dcc.Graph(id="probExceed_cess")),
+                        ],
+                        id="cess_dbct",
+                        label="Cessation Date",
+                    ),
                 ],
-                id="cess_dbct",
-                label="Cessation Date",
-            ),
-        ],
-        className="mt-4",
+                className="mt-4",
+            )
+        
+            # return html.Img(
+            #     style={"width": "600px"},
+            #     src=(
+            #         "https://iridl.ldeo.columbia.edu/dlcharts/render/"
+            #         "905cdac6e87a58586967e115a18e615d01530ddd?_wd=1200px&_ht=600px"
+            #         "&_langs=en&_mimetype=image%2Fpng"
+            #         "&region=bb%3A39.375%3A7.125%3A39.625%3A7.375%3Abb"
+            #         "&waterBalanceCess=3&drySpellCess=10&plotrange2=15"
+            #     ),
+            # )
```

Comment:
Are you planning on using this commented code? I would prefer to see it removed.

Reply:
I don't know what this is. Looks like someone was hoping to print dlchart graph in the app to compare Python with Ingrid/DLchart? It can very likely be removed